docs/source/internals/contributing/running-tests.rst

```diff
@@ -2,15 +2,33 @@
 Test suite
 ==========
 
+Testing requirements
+--------------------
+
+You'll need:
+
+- A running PostgreSQL server
```

Comment:
SQLite also works, through `--sqlite` params. Although couple of tests would not pass.